src/app/Http/Controllers/Api/CommentController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\CommentGetRequest;
+use App\Http\Requests\CommentRequest;
+use App\Http\Resources\CommentResource;
+use App\Models\Comment\Comment;
+use Illuminate\Http\Request;
+
+class CommentController extends Controller
+{
+    public function index(CommentGetRequest $request)
+    {
+        $data = $request->validated();
+        $query = Comment::with('user')->latest()->where('movie_id', $data['movie_id']);
+        return CommentResource::collection($query->paginate());
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CommentRequest $request)
+    {
+        $data = $request->validated();
+        Comment::create([
+            'user_id' => $data['user_id'],
+            'movie_id' => $data['movie_id'],
+            'comment' => $data['text']
+        ]);
+        return response('success', 200);
```

Comment:
Kada radimo store mozes vratiti taj novo kreirani comment i odmah ga zakaciti na movie. Ako ga kaces sa FE je lose jer ne znas dal je taj komentar ok (validacija, autorizacija itd itd) tek kada sa BE dobijes nazad model novo kreiranog komentara onda iskoristis to i zakacis na film